cypress/integration/form-builder/conditionalFieldset.test.js

```diff
@@ -37,6 +37,12 @@ const doc = {
   singleReadOnlyCallbackTrue1: 'Lorem',
 }
 
+const waitForReviewChanges = () => {
```

Comment:
Instead of using this and calling it inside each test, you can probably use the `beforeEach` hook? Like this:

```
 beforeEach(() => {
    cy.contains('Changes', {timeout: 10000}).should('be.visible')
  })
```

Reply:
That makes sense for this test, will update